src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -110,6 +118,19 @@ public static int Run(string[] args)
             }
         }
 
+        private static bool ContainsBuiltTestSources(string[] args)
+        {
+            foreach (var arg in args)
+            {
+                if (!(arg.StartsWith("-") || arg.StartsWith("/")) &&
+                    (arg.EndsWith("dll", StringComparison.OrdinalIgnoreCase) || arg.EndsWith("exe", StringComparison.OrdinalIgnoreCase)))
```

Comment:
shouldn't we use Path.GetExtension() rather than doing ends with

Reply:
https://github.com/microsoft/vstest/blob/master/LICENSE